recipes/pyobjcryst/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "pyobjcryst" %}
+{% set version = "2.0.2" %}
+{% set sha256 = "b48b1a66973dc42d8d15c5ddcf4989cb4160cab220c3a3c60664af46926092d4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy 1.11.2
```

Comment:
@pavoljuhas Do we need to pin to this numpy or can we go higher?

Reply:
Runtime numpy can be more recent; it will work with pyobjcryst build against numpy 1.11.2.  On the other hand, pyobjcryst build against the last numpy may be incompatible with earlier numpy-s.  I don't want pyobjcryst forcing the latest numpy on its target environment.

Please, keep the build-stage numpy as is, but use `numpy >=1.11.2` for the `run` requirements.

Reply:
Not a big deal, but what about `numpy 1.11.*,>=1.11.2` assuming the `.2` patch release is necessary to build?